packaging/opensuse-42.2/snapd.spec

```diff
@@ -149,6 +149,11 @@ go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
 %gobuild cmd/snapctl
 %gobuild cmd/snap-update-ns
 
+# Remove static link to seccomp as it has been disabled and to avoid ld error
+sed -e "s/-Bstatic -lseccomp/-Bstatic/g" -i %{_builddir}/go/src/%{provider_prefix}/cmd/snap-seccomp/main.go
```

Comment:
Can you please expand on the comment. Why do we need to disable static linking of libseccomp, what is the error?